superset-frontend/src/pages/ChartList/index.tsx

```diff
@@ -287,9 +287,7 @@ function ChartList(props: ChartListProps) {
       ...filters,
     });
     const response: void | JsonResponse = await SupersetClient.get({
-      endpoint: !filterValue
-        ? `/api/v1/dashboard/?q=${queryParams}`
-        : `/api/v1/chart/?q=${queryParams}`,
+      endpoint: `/api/v1/dashboard/?q=${queryParams}`,
```

Comment:
@michael-s-molina I briefly looked into this logic previously. Could you confirm that your logic also works when you include a `filterValue`. i.e., text in the `SEARCH` box?

Reply:
If you use the search input (not the dashboard select) to look for a dashboard name, it won't work. That would require creating a [search filter](https://github.com/apache/superset/blob/83ff4cd86a4931fc8eda83aeb3d8d3c92d773202/superset/charts/filters.py#L133) but that's out of the scope of this PR.